tests/e2e/utils/common_utils.go

```diff
@@ -0,0 +1,16 @@
+package utils
+
+import (
+	"io/ioutil"
+)
+
+func ReadFile(path string) ([]byte, error) {
```

Comment:
I think we can remove this? This function doesn't do anything. Can remove later after merge. Thoughts?

Reply:
I see elsewhere we are using os.ReadFile as well which seems to do the same thing.

Reply:
Looks like os.ReadFile is more direct over io/ioutil now... at least from reading https://pkg.go.dev/io/ioutil@go1.17.6#ReadFile

Reply:
@kaovilai I do think the same. I just kept it as it is, considering these changes were made by some earlier PR. I can definitely try to replace it altogether with os.ReadFile() if possible.